roles/overpass-api/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+user: overpass
```

Comment:
Should it be overpass_user instead of user ?
I'm not sure if it is possible to set a parameter for a host and a single role with ansible.